src/crypto/sha256.h

```diff
@@ -20,8 +20,10 @@ class CSHA256 {
     static const size_t OUTPUT_SIZE = 32;
 
     CSHA256();
+    CSHA256(const unsigned char iv[OUTPUT_SIZE]);
     CSHA256 &Write(const uint8_t *data, size_t len);
     void Finalize(uint8_t hash[OUTPUT_SIZE]);
+    void Midstate(unsigned char hash[OUTPUT_SIZE], uint64_t* bytes, unsigned char* buffer);
```

Comment:
second parameter... is it `bytes` or is it `len`? one thing in the header, another in the implementation.